### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by mwtoews** [csdms/bmi-python#19]
A new release on PyPI should be published, since the most recent is [2.0 from December 2019](https://pypi.org/project/bmipy/2.0/). This will resolved a pip warning:
> DEPRECATION: bmipy is being installed using the legacy 'setup.py install' method, because it does not have a 'pyproject.toml' and the 'wheel' package is not installed. pip 23.1 will enforce this behaviour change.

This was already resolved by #12

**Comment by mcflugen** [comment]
@mwtoews Thanks for the reminder. Yes, we definitely should get a new release out. #20 cleans up the repository and once that's merged we'll publish a new release.

 Out of curiosity, are you a user of any CSDMS products like BMI or *Landlab*? If so, we would love to hear what you're doing.

**Comment by mwtoews** [comment]
@mcflugen Thanks for update. I'm not using Landlab, but I'll take a look for curiosities-sake. My present focus is with hydrology models; see [doi:10.1016/j.envsoft.2021.105257](https://doi.org/10.1016/j.envsoft.2021.105257) using [Deltares/xmipy](https://github.com/Deltares/xmipy) and [MODFLOW-USGS/modflowapi](https://github.com/MODFLOW-USGS/modflowapi)

**Comment by wpbonelli** [comment]
Hi, I noticed that `versioneer.py` in the v2.0 release breaks install on Python 3.12 since `SafeConfigParser` was finally [removed from `configparser`](https://docs.python.org/3/whatsnew/3.12.html#configparser).

I see #18 removes versioneer and I know v2.0 only supports up to py3.11 anyway. Just a heads up for others, we ran into it on the [modflow6 repo](https://github.com/MODFLOW-USGS/modflow6/actions/runs/6483165570/job/17615572280#step:4:491) because we didn't specify a python version for the `setup-micromamba` action.
